pages/dkp/kommander/2.2/install/air-gapped/index.md

```diff
@@ -183,7 +183,7 @@ export VERSION=v2.2.0
     dkp push image-bundle --image-bundle kommander-image-bundle.tar.gz --to-registry <REGISTRY_URL>
     ```
 
-Based on the network latency between the environment of script execution and the docker registry, this can take a while to push all the images to your image registry.
+It may take a while to push all the images to your image registry, depending on the performance of the network between the machine you are running the script on and the docker registry.
```

Comment:
```suggestion
It may take a while to push all the images to your image registry, depending on the performance of the network between the machine you are running the script on and the Docker registry.
```